core/browser_events.ts

```diff
@@ -151,13 +151,17 @@ export function bind(
 
     // Add equivalent touch event.
     if (name in Touch.TOUCH_MAP) {
-      const touchWrapFunc = (e: AnyDuringMigration) => {
+      const touchWrapFunc = (e: Event) => {
         // Punt on multitouch events.
-        if (e.changedTouches && e.changedTouches.length === 1) {
+        if (e instanceof TouchEvent && e.changedTouches &&
+            e.changedTouches.length === 1) {
           // Map the touch event's properties to the event.
           const touchPoint = e.changedTouches[0];
-          e.clientX = touchPoint.clientX;
-          e.clientY = touchPoint.clientY;
+          // TODO: We are trying to make a touch event look like a mouse event,
```

Comment:
Need feedback on this.

Reply:
I feel like we should probably introduce our own common event, or maybe see if we can use PointerEvent? But fine to leave for a later PR I think.